enhancements/cluster-logging/cluster-logging-otel-support.md

```diff
@@ -0,0 +1,240 @@
+1. Deploys the Red Hat **cluster-logging-operator**
+1. Creates a **ClusterLogForwarder** custom resource for the **LokiStack**
+    1. **NOTE:** In Tech Preview, this resources includes an annotation to instruct the operator to use the OTEL datamodel in lieu of ViaQ
+
+The **cluster-observability-operator**:
+
+1. Deploys the console **Logging UIPlugin** for reading logs in the OpenShift console
+
+The **loki-operator**
+
+1. Deploys the **LokiStack** for storing logs in-cluster
+
+The **cluster-logging-operator**:
+
+1. Deploys the log collector to forward logs to log storage in the `openshift-logging` namespace
+
+
+### API Extensions
+
+During the time this feature is considered Tech Preview, instances of **ClusterLogForwarder** will be [annotated](./forwarder-to-otlp.md) to enable
+forwarding to OTLP receivers. Additionally, a field will be added to the **LokiStack** output type where administrators can configure the desired
+data model of forwarded logs.
+
+Example of configuring multiple outputs:
+
+```yaml
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+spec:
+  outputs:
+  - name: my-loki-otel
+    type: lokiStack
+    lokiStack:
+      dataModel: otel
+  - name: my-other-loki-otel
+    type: lokiStack
+    lokiStack:
+      dataModel: otel
+  - name: my-legacy-loki                  #dataModel: viaq
+    type: lokiStack
+  pipelines:
+  - name: in-to-out
+    outputRefs: [my-loki-otel,my-other-loki-otel,my-legacy-loki]
+    inputRefs: [application]
+```
+
+This example configures a collector to forwards logs to multiple LokiStacks (e.g. my-loki-otel,my-other-loki-otel) using OTEL semantic attributes while
+also forwarding logs to a separate LokiStack (e.g. my-legacy-loki) using ViaQ.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+#### Standalone Clusters
+
+#### Single-node Deployments or MicroShift
+
+### Implementation Details/Notes/Constraints
+
+#### Log Storage
+
+* The **loki-operator** will be updated so deployments of Lokistack rely upon the OTLP API instead of the Push API. This will allow
+trusted and verified Red Hat products, like Red Hat Build of the OpenTelemetry Collector, to forward logs to the same LokiStack as the Red Hat OpenShift Logging.  Additionally, this new version of the **loki-operator**
+will support server-side indexing to improve query performance and simplify configuration on the client side. Sane defaults will be provided for users using the `openshift-logging` tenancy.
+* The **loki-operator** will configure Loki deployments with server-side indexing of OTEL and Minimal ViaQ Product Labels (MVP).  The Red Hat Observability Logging 
+[Data Model](https://github.com/rhobs/observability-data-model/pull/6)
+provides additional details.
+
+#### Log Visualization
+
+The **Logging UIPlugin** will be modified to replace its UI widgets to rely upon OTEL attributes in lieu of ViaQ.  These changes will be
```

Comment:
These changes will be released to production one release after the other components achieve **GA**.   can you explain this sentence?  I think Logging visualization must be ready when the other components achieve **GA**.